src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.scss

```diff
@@ -13,31 +17,35 @@
   }
 }
 
-.previouslyViewedType {
+.objectType {
   color: $dark-grey;
   margin-left: 8px;
   font-size: 12px;
   font-weight: 300;
 }
 
+.objectId {
+  margin-left: 8px;
+}
+
 .title {
   font-size: 14px;
-  margin: 5px 0 10px 10px;
+  margin: 5px 0 25px 0;
   font-weight: $bold;
 }
 
-.sectionTitle {
-  color: $dark-grey;
+.more {
   font-size: 12px;
-  border-bottom: 1px solid $grey;
-  margin-bottom: 15px;
   position: relative;
-  margin-top: 20px;
+  margin-top: 25px;
+  display: inline-flex;
+  cursor: pointer;
+  color: $blue;
+  font-weight: $normal;
 
   .ellipsis {
```

Comment:
1) Why are we using an svg ellipsis here rather than a regular unicode ellipsis?
2) If we switch ellipsis to a regular text ellipsis, the `.more` class will have no need for `display: inline-flex`